DesignAutomationHandler2020/packages.config

```diff
@@ -0,0 +1,5 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<packages>
+  <package id="Autodesk.Forge.DesignAutomation.Revit" version="2020.0.0-beta1" targetFramework="net472" />
+  <package id="Microsoft.CSharp" version="4.5.0" targetFramework="net472" />
```

Comment:
Just want to double check here. Was 2020 DesignAutomation Bridge published?

Reply:
@zhuliice Yes.
https://www.nuget.org/packages/Autodesk.Forge.DesignAutomation.Revit/2020.0.0-beta1